cypress/support/validations/schemaValidation.js

```diff
@@ -152,22 +147,45 @@ Cypress.Commands.add(
   * @param {Object} response - JSON response string
   * @param {string} methodOrEvent - String containing the method/event in the format "<Module.Method>" or "<Module.Event>"(Ex: accessibility.closedCaptionsSettings, 'accessibility.onClosedCaptionsSettingsChanged')
   * @param {*} params - API params
-  * @param {string} sdkVersion - SDK version
   * @param {string} schemaType - schema type determines which schema should fetch result/error
+  * @param {boolean} isValidation - flag to determine if response is updated as part of invocation or validation glue. For former case, flag is set to false by default, else for latter case flag is set to true
   * @example
```

Comment:
Not able to understand the purpose of **_isValidation_** here. In all cases, we can make use of validation present inside FCS and not during invocation

Reply:
As discussed in call